programs/reward-pool/src/lib.rs

```diff
@@ -0,0 +1,1111 @@
+use borsh::{BorshDeserialize, BorshSerialize};
```

Comment:
Overall, do you think it would be beneficial to put the smart contract code and tools in a dedicated repository rather than in the backend, as it will not be necessary to publish it on the backend servers?